src/SwedbankPay.Sdk/LinkRelation.cs

```diff
@@ -57,7 +59,7 @@ public static readonly LinkRelation
         public static readonly LinkRelation PaidPayment = new LinkRelation(nameof(PaidPayment), PaymentResourceOperations.PaidPayment);
 
         public static readonly LinkRelation UpdatePaymentAbort =
-            new LinkRelation(nameof(UpdatePaymentAbort), PaymentResourceOperations.UpdatePaymentAbort);
+            new LinkRelation(nameof(UpdatePaymentAbort), PaymentResourceOperations.Abort);
```

Comment:
For consistency I think `UpdatePaymentAbort` is correct at this level.

```suggestion
            new LinkRelation(nameof(UpdatePaymentAbort), PaymentResourceOperations.UpdatePaymentAbort);
```